src/lib/transaction_logic/test/account_timing.ml

```diff
@@ -0,0 +1,194 @@
+open Core_kernel
+open Mina_transaction_logic
+open Currency
+open Mina_base
+open Mina_numbers
+
+type timing = (Global_slot.t, Balance.t, Amount.t) Account_timing.tt
```

Comment:
isn't this the same as `Account_timing.t`?

Reply:
I'm eliminating `timing` alias. Deleting `Account_timing.tt` requires updating many other modules though, so I'd refrain myself from doing it now. Besides, I'm not even sure it's desired. I certainly prefe to be able to write `Account.Timing.Untimed` rather than `Account.Timing.Poly.Untimed`. Not to mention the fact that when `Account.Timing` is open, seeing `Poly.Untimed` somewhere in the code might be quite confusing, as there's a lot of these `Poly` modules around. If anything, I'd get rid of the type altogether, leaving only the record inside it as an optional field on `Account.t`. The fact that the record is inlined can be very annoying when trying to access its fields. But that's certainly a change for another time too.